symfony/framework-bundle/4.2/public/index.php

```diff
@@ -4,9 +4,7 @@
 use Symfony\Component\Debug\Debug;
 use Symfony\Component\HttpFoundation\Request;
 
-require dirname(__DIR__).'/vendor/autoload.php';
-
-Kernel::bootstrapEnv();
+require dirname(__DIR__).'/src/.bootstrap.php';
 
 if ($_SERVER['APP_DEBUG']) {
```

Comment:
How about moving this block to the bootstrap file as it is currently duplicated in public/index.php and bin/console

Reply:
This would also enable the debug stuff for phpunit when enabled. Is that a problem?

Reply:
Yes, that'd be a problem, so we cannot do that (I did it in an earlier version and reverted for this reason.)